api/specs.proto

```diff
@@ -186,6 +186,9 @@ message ContainerSpec {
 	// including stdin if it is still open.
 	bool tty = 13 [(gogoproto.customname) = "TTY"];
 
+	// interactive declares that the standard input (stdin) should be open.
```

Comment:
Could we remove the commentary about "interactive" and only document that this flag keeps stdin open?